environment/src/test/java/com/sequenceiq/environment/environment/service/EnvironmentCreationServiceTest.java

```diff
@@ -102,6 +102,7 @@
     void setup() {
         when(validatorService.validatePublicKey(any())).thenReturn(ValidationResult.empty());
         when(validatorService.validateTags(any())).thenReturn(ValidationResult.empty());
+        when(validatorService.validateEncryptionKeyUrl(any(), any())).thenReturn(ValidationResult.empty());
     }
```

Comment:
Could you please add a testcase for the unhappy scenario, i.e. when `validatorService.validateEncryptionKeyUrl()` returns a failure?